datafusion/core/src/sql/planner.rs

```diff
@@ -3458,7 +3458,8 @@ mod tests {
         let expected = "Projection: #person.first_name, #person.id\
         \n  Inner Join: Using #person.id = #person2.id\
         \n    TableScan: person projection=None\
-        \n    TableScan: person2 projection=None";
+        \n    SubqueryAlias: person2\
```

Comment:
the appearance of `SubqueryAlias` in this query is  somewhat strange to me as there is no subquery in this SQL (maybe a better name is `RelationAlias` or something)?

Reply:
I originally named this `AliasedRelation` and then changed it to `SubqueryAlias` because that is what Spark uses. I don't have a strong preference either way.

Reply:
me neither. Whatever you prefer is fine with me